docs/docs/contribute/software/add-new-metric-provider.md

```diff
@@ -0,0 +1,100 @@
+# Adding a New Metrics Provider for Dummy Endpoint
+
+To create a provider for the dummy endpoint, follow these steps:
+
+1. Fork the [Keptn repo](https://github.com/keptn/lifecycle-toolkit)
+
+2. **Define the Provider Type:** In the `metrics-operator/controllers/common/providers/common.go` file,
+ define the constant `KeptnDummyProviderType` with the value `"dummy"`.
+
+    ```go
+    const KeptnDummyProviderType = "dummy"
+    ```
+
+3. **Implement the Provider:** Create your own new folder inside the
+[metrics-operator/controllers/common/providers](https://github.com/keptn/lifecycle-toolkit/tree/main/metrics-operator/controllers/common/providers)
+ matching the new service name: dummy and a new Go package for the dummy provider.
+  This package should contain
+ a struct that implements the `KeptnSLIProvider` interface.
+  To fully implement the `KeptnSLIProvider` interface, it's necessary to implement the following functions.
+  `EvaluateQuery`(Fetches metric values from the dummy provider)
+   - This function fetches metric values based on the provided
+     metric query from the dummy provider.
+     It evaluates the query and returns the metric values
+     along with any additional data if required.
+  `EvaluateQueryForStep`(Fetches metric values with step interval from the dummy provider)
```

Comment:
```suggestion
  `EvaluateQueryForStep`(Fetches metric values with step interval from the provider)
```

Reply:
I would not write dummy in this sections it is true for any interface

Reply:
True